BibBuddy/app/src/androidTest/java/de/bibbuddy/NotesFragmentUITest.java

```diff
@@ -19,191 +22,173 @@
-
-        // onView(withId(R.id.noteType)).check(matches(withId(R.drawable.microphone));
-        // onView(withId(R.id.noteType)).check(matches(withId(R.drawable.picture)));
-
-    }
-
-    // TODO: Needs to be adjusted as soon as more note types are added
-    @Test
-    public void addButtonAndAddMenu_AreShown() {
-        onView(withId(R.id.addButton)).check(matches(isDisplayed()));
-        onView(withId(R.id.addButton)).perform(click());
-        onView(withText(R.string.create_text_note)).check(matches(isDisplayed()));
-        onView(withText(R.string.create_voice_note)).check(matches(isDisplayed()));
-        onView(withText(R.string.take_picture_note)).check(matches(isDisplayed()));
-    }
-
-    // TODO: Needs to be adjusted as soon as more note types are added
-    @Test
-    public void Editor_IsOpened_OnItemClick() {
-        ViewActions.closeSoftKeyboard();
-        onView(withId(idText)).perform(click());
-        onView(withId(R.id.fragment_text_note_editor)).check(matches(isDisplayed()));
-
-    */
-/*
-    onView(withId(id2)).perform(click());
-    onView(withId(nF.getActivity().getSupportFragmentManager().findFragmentById(R.id.fragment_voice_note_editor).getId())).check(matches(isDisplayed()));
-    onView(withId(id3)).perform(click());
-    onView(withId(nF.getActivity().getSupportFragmentManager().findFragmentById(R.id.fragment_picture_note_editor).getId())).check(matches(isDisplayed()));
-    *//*
-
-
+  static NotesFragment nF;
+  private static View itemView;
+  private final String exampleText = "text";
+  private String modDate;
+  private String name;
+  private int idText;
+  private int idVoice;
+  private int idPicture;
+
+  @Before
+  public void init() {
+    ActivityScenario scenario = ActivityScenario.launch(MainActivity.class);
+    onView(withId(R.id.navigation_notes)).perform(click());
+    onView(withId(R.id.fragment_notes)).check(matches(isDisplayed()));
+    ActivityScenario activityScenario = scenario.onActivity(
+        (ActivityScenario.ActivityAction<MainActivity>) activity -> {
+          nF = (NotesFragment) activity.getSupportFragmentManager().getFragments().get(0);
+          NotesFragment.notes.clear();
+          if (NotesFragment.notes.size() == 0) {
+            Note textNote =
+                new Note(exampleText, 0, exampleText);
+            modDate = String.valueOf(textNote.getModDate());
+            name = textNote.getName();
+            NoteTextItem noteTextItem = new NoteTextItem(modDate, name, textNote.getId());
+            NotesFragment.notes.add(noteTextItem);
+          }
+        });
+
+    onView(ViewMatchers.withId(R.id.recyclerView))
+        .perform(RecyclerViewActions.scrollTo(hasDescendant(withText(exampleText))));
+    RecyclerView recyclerView =
+        Objects.requireNonNull(nF.getView()).findViewById(R.id.recyclerView);
+    itemView = recyclerView.getChildAt(0);
+    idText = itemView.getId();
+  }
+
+  @Test
+  public void NotesFragmentDisplay_Test() {
```

Comment:
There is a message from Java Checkstyle:
`Method name 'NotesFragmentDisplay_Test' must match pattern '^[a-z][a-z0-9][a-zA-Z0-9_]*$'.
`

Reply:
Done - you can not use _Test (which we wanted to use after a quick discussion) - I just deleted the _ .